metrics/internal/collectors/storageconsumer.go

```diff
@@ -162,3 +162,16 @@ func getAllStorageConsumers(lister StorageConsumerLister) (storageConsumers []*o
 	}
 	return
 }
+
+func (c *StorageConsumerCollector) GetClientId() string {
+	storageConsumerLister := NewStorageConsumerLister(c.Informer.GetIndexer())
```

Comment:
Do we have to use a lister and list? why can't we just load the correct consumer (based on its name 'internal') directly?